docs/index.html

```diff
@@ -15,6 +15,7 @@
                 <h1>Garage Benchmarking</h1>
                 <br/>
                 <p> Time last updated: <small id="time"></small> </p>
+                <p> Ran from commit: <a id="githash"> link </a> </p>
```

Comment:
can you actually use the hash itself for the link? it is useful to be able to easily copy it to check out that revision from a terminal.

Reply:
<img width="757" alt="Screen Shot 2019-05-14 at 2 55 34 PM" src="https://user-images.githubusercontent.com/9795673/57735105-61b59300-7658-11e9-9add-8b9d55c759c1.png">